bigchaindb/db/utils.py

```diff
@@ -18,54 +18,91 @@ def get_conn():
                      bigchaindb.config['database']['port'])
 
 
-def init():
-    # Try to access the keypair, throws an exception if it does not exist
-    b = bigchaindb.Bigchain()
+def get_database_name():
+    return bigchaindb.config['database']['name']
 
-    conn = get_conn()
-    dbname = bigchaindb.config['database']['name']
 
+def create_database(conn, dbname):
     if r.db_list().contains(dbname).run(conn):
         raise exceptions.DatabaseAlreadyExists('Database `{}` already exists'.format(dbname))
 
     logger.info('Create:')
```

Comment:
@sbellem, @r-marques: since we are refactoring, I'm not sure the usage of `log` here is the best. There is no "best practice" for that, but my gut tells me that we are not logging single messages, but a group of messages that together makes sense, separated no. A log message should be about an event, and should make sense by itself, while here we have sparse information, so instead of:
```
 - database `%s`
```
something like:
```
Create database `%s`
```
is better IMO.

What do you think?